package.json

```diff
@@ -1,14 +1,23 @@
 {
   "name": "gtoken",
-  "version": "1.2.3",
+  "version": "2.0.0",
```

Comment:
The version should only be bumped at release/publish time.

Reply:
fixed